mat-process-api/V1/Controllers/ProcessImageController.cs

```diff
@@ -78,6 +78,11 @@ public IActionResult GetProcessImage([FromRoute] GetProcessImageRequest requestD
                     var usecaseResponse = _processImageUseCase.ExecuteGet(requestData);
                     return Ok(usecaseResponse);
                 }
+                catch(ImageNotFound ex)
+                {
+                    return NotFound($"The image with ID = {requestData.imageId} has not been found.");
+                    throw ex;
```

Comment:
Code will never get to this **throw** statement due to the **return** statement.